octoprint_mrbeam/static/js/mother_viewmodel.js

```diff
@@ -205,7 +208,16 @@ $(function () {
             self.removeClassesForModifierKeys = function (event) {
                 if (event.which === 16)
                     document.body.classList.remove("shiftKey");
-                if (self.KEY_CODES_CTRL_COMMAND.includes(event.which))
+                if (
+                    self.KEY_CODES_CTRL_COMMAND.includes(event.which) ||
+                    event.type == "contextmenu"
+                )
```

Comment:
I think it is best if the "contextmenu" events have the same effect as "keyup" for all the keys -> 
```js
$("body").on("contextmenu", function (event) {
    ["shiftKey", "altKey", "ctrlKey"].foreach(document.body.classList.remove);
});
```

Reply:
as far as we know, this issue is very specific to mac and how the ctrl key opens a context menu.